doc/supported_mimetypes.rst

```diff
@@ -9,14 +9,12 @@ Here is an overview of supported file formats:
 +==================================================================================+=============+
 | **Images - based on WAND (image magick)**                                                      |
 +----------------------------------------------------------------------------------+-------------+
-| application/postscript                                                           | .ps         |
+| application/postscript                                                           | .ai         |
```

Comment:
not an error, both .ai and .ps have this mimetype, automatic script get just the first/ the last one. We should later the  fix this to show all file extension instead of just one.

Reply:
just created issue about it : https://github.com/algoo/preview-generator/issues/165

Reply:
script generation modified